geode-core/src/main/java/org/apache/geode/cache/partition/PartitionRebalanceInfo.java

```diff
@@ -137,4 +137,11 @@
    * @return the time, in milliseconds, spent transferring primaries for this region
    */
   long getPrimaryTransferTime();
+
+  /**
+   * Returns the total number of members on which command is executed.
+   *
+   * @return the total number of members on which command is executed
+   */
+  int getPrimaryNumOfMembers();
```

Comment:
This name is confusing, since the method has nothing to do with primary buckets. A better name might be "getNumberOfMembersExecutedOn".